lumen/ai/agents.py

```diff
@@ -376,7 +379,7 @@ async def answer(self, messages: list | str):
             for source in available_sources:
                 for table in source.get_tables():
                     tables_to_source[table] = source
-                    schema = get_schema(source, table, include_min_max=False, include_enum=False, limit=1)
+                    schema = get_schema(source, table, include_min_max=False, include_enum=True, limit=1)
```

Comment:
An enum can give a bit more context, especially if the column names are ambiguous.

<img width="582" alt="image" src="https://github.com/user-attachments/assets/a9a789d3-7823-4a3f-8742-c15794a6f06a">